### ISSUE TITLE
the losses function

### ISSUE THREAD

**Issue by Mrils** [balancap/SSD-Tensorflow#230]
Hello everyone! 
I'm training the net using PASCAL VOC 2012 dataset and find that the total loss is always higher than 40. Then I begin to observe the loss of every block using Tensorboard, the neg_loss is dropping but the pos_loss and loc_loss are so strange.
Why? What's wrong with the code?

**Comment by EricZgw** [comment]
I also encountered the same problem. Did you solve it?

**Comment by jhgfkdj** [comment]
@EricZgw Hi I got this problem too, and my loss got a violent vibration, never declined under 100. Its so weird~~

**Comment by sunrise666** [comment]
try to change optimizer and learning_rate and weight_decay,use tensorboard to see how the value affect your loss

**Comment by chenjjjjuu** [comment]
> @EricZgw Hi I got this problem too, and my loss got a violent vibration, never declined under 100. Its so weird~~

me too ,my loss even up to 200+.
